github_run.py

```diff
@@ -118,17 +118,19 @@ def new_pr(self, not_posted):
             metadata = yaml.safe_load(metadata)
             if media:
                 metadata["media"] = media
-                metadata["mentions"] = {
-                    key: value
-                    for key, value in metadata.get("mentions", {}).items()
-                    if key in media
-                }
-                metadata["hashtags"] = {
-                    key: value
-                    for key, value in metadata.get("hashtags", {}).items()
-                    if key in media
-                }
-            new_md_content = f"---\n{yaml.dump(metadata)}---\n{text}"
+                if "mentions" in metadata:
+                    metadata["mentions"] = {
+                        key: value
+                        for key, value in metadata["mentions"].items()
+                        if key in media
+                    }
```

Comment:
```suggestion
                    for key, value in metadata["mentions"].items():
                        if key in media:
                            metadata["mentions"].update({key: value})
```

or something like that, please ;)

Reply:
but I think that might be different from what the other code is doing.
It is supposed to only keep the mentions and hashtags that are part of specific media.